inc/geolocation.class.php

```diff
@@ -249,6 +254,11 @@ public static function addDefaultWhere() {
       return $where;
    }
 
+   /**
+    * Dislay the agents according the datetime
```

Comment:
A typo in Display. Also, in other methods you don't use imperative form. If I rely on your preferred descriptions you should say "Displays (final s)".